storage/reads/array_cursor.gen.go.tmpl

```diff
@@ -235,6 +236,61 @@ func (c *integer{{.Name}}CountArrayCursor) Next() *cursors.IntegerArray {
 	}
 }
 
+type integer{{.Name}}WindowCountArrayCursor struct {
+	cursors.{{.Name}}ArrayCursor
+	every int64
+	tr    datatypes.TimestampRange
+}
+
+func (c *integer{{.Name}}WindowCountArrayCursor) Stats() cursors.CursorStats {
+	return c.{{.Name}}ArrayCursor.Stats()
+}
```

Comment:
I'm just curious because its not super clear to me, what are these `CursorStats` tracking here?

Reply:
`CursorStats` tracks the number of values and the number of bytes read. I'm not sure how it's used at a higher level.